### PR TITLE
[ADD] trader microsite-statistics-screen

### PR DESCRIPTION
**Pull request by ekediala**
Added screen for trader to view their microsite summary report

PS: Please do not forget to add `app.json` before testing.

PPS: Changes count was inflated by the downloaded fonts.

### CODE REVIEW

**Comment by parkerthegeniuschild** [comment]
does this pull in live data already?

File: src/components/viewMicrosite/components/TraderViewMicrosite.jsx [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+              strokeWidth: "2",
+              stroke: "#ffa726"
+            }
+          }}
+          bezier
+          style={{
+            borderRadius: 16,
+            margin: 5,
+            marginTop: 70
+          }}
```
Comment (mrayor) [inline src/components/viewMicrosite/components/TraderViewMicrosite.jsx]: inline styles

File: src/components/viewMicrosite/components/TraderViewMicrosite.jsx [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+                  Math.random() * 100,
+                  Math.random() * 100,
+                  Math.random() * 100,
+                  Math.random() * 100,
+                  Math.random() * 100,
+                  Math.random() * 100
+                ]
+              }
+            ]
+          }}
```
Comment (mrayor) [inline src/components/viewMicrosite/components/TraderViewMicrosite.jsx]: I think you should put this dummy data in a variable or a separate file till we have real data to pull in.

File: src/components/viewMicrosite/components/TraderViewMicrosite.jsx [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+              r: "6",
+              strokeWidth: "2",
+              stroke: "#ffa726"
+            }
+          }}
+          bezier
+          style={{
+            borderRadius: 16,
+            margin: 5,
+            marginTop: 70
```
Comment (mrayor) [inline src/components/viewMicrosite/components/TraderViewMicrosite.jsx]: I can see dollars, some inline styles, and some values that are best to be put in a constant file, e.g the naira symbol in the y axis label. The y axis suffix is also hardcoded but i assume that would change when we pull in real data right?. I'll suggest you put it in the `components/common/constant` so it easily accessible anywhere in the app.

File: src/components/viewMicrosite/components/TraderViewMicrosite.jsx [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+            <Text style={Object.assign({}, tabStyle, sixMonths)}>
+              {SIX_MONTHS}
+            </Text>
+          </View>
+
+          <View>
+            <Text style={Object.assign({}, tabStyle, oneYear)}>
+              {ONE_YEAR}
+            </Text>
+          </View>
```
Comment (mrayor) [inline src/components/viewMicrosite/components/TraderViewMicrosite.jsx]: You are repeating yourself a lot here...You should map over these.

File: src/components/viewMicrosite/components/TraderViewMicrosite.jsx [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+
+const TraderViewMicrosite = () => {
+  const state = useSelector(state => state.login.user);
+  const { businessName } = state;
+  return (
+    <>
+      <View style={main}>
+        <View style={drawerIcon}>
+          <HeaderIcon onPress={() => null} />
+        </View>
```
Comment (mrayor) [inline src/components/viewMicrosite/components/TraderViewMicrosite.jsx]: Please use the react-navigation drawer to sort out the drawer navigations. At the moment, there is a back button that takes people to create microsite.

File: src/components/viewMicrosite/components/TraderViewMicrosite.jsx [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+              height="63"
+            />
+            <View>
+              <Text style={labelText}>Reviews</Text>
+            </View>
+
+            <View>
+              <Text style={labelCount}>0</Text>
+            </View>
+          </SVG>
```
Comment (mrayor) [inline src/components/viewMicrosite/components/TraderViewMicrosite.jsx]: I think you are repeating yourself all over here as well.

File: src/components/viewMicrosite/components/TraderViewMicrosite.jsx (hunk 3 above)
Comment (ekediala) [reply src/components/viewMicrosite/components/TraderViewMicrosite.jsx]: Yup.

File: src/components/viewMicrosite/components/TraderViewMicrosite.jsx (hunk 5 above)
Comment (ekediala) [reply src/components/viewMicrosite/components/TraderViewMicrosite.jsx]: I want to put that in a different PR. This is already big enough.

Comment (mrayor) [reply src/components/viewMicrosite/components/TraderViewMicrosite.jsx]: You should remove this then...its not needed.

**Comment by mrayor** [comment]
LGTM

**Pull request merged**